server/src/test/java/org/apache/druid/client/BrokerInternalQueryConfigTest.java

```diff
@@ -56,7 +58,7 @@ public void testSerde() throws Exception
         BrokerInternalQueryConfig.class
     );
 
-    Assert.assertEquals(ImmutableMap.of(), config.getContext());
```

Comment:
Haven't gone through the whole PR yet, just a drive-by comment.
Druid code generally follows the style of not have static imports, rather qualifying the static method calls by the respective class names. So this would be:

```
Assertions.assertEquals(...)
```